src/docs/10_quality_scenarios.adoc

```diff
@@ -3,61 +3,16 @@
 
 
 [role="arc42help"]
-****
-
-.Content
-This section contains all quality requirements as quality tree with scenarios. The most important ones have already been described in section 1.2. (quality goals)
-
-Here you can also capture quality requirements with lesser priority,
-which will not create high risks when they are not fully achieved.
-
-.Motivation
-Since quality requirements will have a lot of influence on architectural
-decisions you should know for every stakeholder what is really important to them,
-concrete and measurable.
-****
+After we discussed some more the design and the goals of this proyect some quality attributes that aren't as notable or important that the ones that we are focusing on came to light. Some of this includes attributes like usability, lernability or flexibility. This three, for example, constitute an integral part of every application and will be taken into consideration in a future stage of development.
```

Comment:
~Lernability~ Learnability.

Reply:
I changed that already as it is just a misspelling